src/main/java/kitchenpos/application/MenuGroupService.java

```diff
@@ -1,26 +1,26 @@
 package kitchenpos.application;
 
-import kitchenpos.dao.MenuGroupDao;
 import kitchenpos.domain.MenuGroup;
+import kitchenpos.repository.MenuGroupRepository;
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
 
 import java.util.List;
 
 @Service
 public class MenuGroupService {
-    private final MenuGroupDao menuGroupDao;
+    private final MenuGroupRepository menuGroupRepository;
 
-    public MenuGroupService(final MenuGroupDao menuGroupDao) {
-        this.menuGroupDao = menuGroupDao;
+    public MenuGroupService(final MenuGroupRepository menuGroupRepository) {
+        this.menuGroupRepository = menuGroupRepository;
     }
 
     @Transactional
     public MenuGroup create(final MenuGroup menuGroup) {
-        return menuGroupDao.save(menuGroup);
+        return menuGroupRepository.save(menuGroup);
     }
 
     public List<MenuGroup> list() {
```

Comment:
@Transactional(readOnly = true) 를 사용하면 어떠한 이점이 있을까요? 😄 
```suggestion
    @Transactional(readOnly = true)
    public List<MenuGroup> list() {
```

Reply:
JPA에 readOnly hint를 전달하게 됩니다!

트랜잭션이 종료될 때 flush가 일어나지 않아서
의도치 않은 업데이트 쿼리가 나가지 않게 하고
변경감지 비용이 발생하지 않아 성능 상 이점이 있습니다😀

readOnly 힌트를 빠트렸네요! 감사합니다🙇🏻‍♂️